nautobot/circuits/views.py

```diff
@@ -102,6 +103,28 @@ class ProviderUIViewSet(NautobotUIViewSet):
     queryset = Provider.objects.all()
     serializer_class = serializers.ProviderSerializer
     table_class = tables.ProviderTable
+    object_detail_content = ObjectDetailContent(
```

Comment:
Should be possible to rip most content out of the `provider_retrieve.html` template now. (Leave the skeleton of the template in case an App is providing a custom tab and still using the base template)